openforcefield/tests/test_molecule.py

```diff
@@ -301,9 +302,12 @@ def test_to_from_smiles(self, molecule, toolkit):
         toolkit_wrapper = toolkit()
 
         undefined_stereo = molecule.name in undefined_stereo_mols
+        # Since OpenEye did the original reading of MiniDrugBank, if OPENEYE doesn't
+        # think a feature is stereogenic, then "molecule" won't have stereochemistry defined
+        stereogenic_in_rdk_but_not_openeye = molecule.name in drugbank_stereogenic_in_rdkit_but_not_openeye
 
         smiles1 = molecule.to_smiles(toolkit_registry=toolkit_wrapper)
-        if undefined_stereo:
+        if undefined_stereo or stereogenic_in_rdk_but_not_openeye:
```

Comment:
This may be over-engineering, but wouldn't be want to fall back to handling stereochemistry if it's using RDKit, which would find it?

```suggestion
        if undefined_stereo or (stereogenic_in_rdk_but_not_openeye && toolkit != 'RDKitToolkitWrapper'):
```

Reply:
Awesome attention to detail. I've switched this over in 9d2e373